File: Level 26/Homework/HW6.py
# 6) მომხმარებელს შემოაყვანინე რიცხვები, შექმენი ორი სია დადებითი და უარყოფითი სიებისთვის, 
# დადებითი რიცხვები დაამატე დადებითი რიცხვებისთვის განკუთვნილ სიაში, უარყოფითი რიცხვები კი პირიქით


positive_numbers = []
negative_numbers = [] 

user_input = input("Enter number or 'stop': ")

while user_input != "stop":
    num = int(user_input)

    if num > 0:
        positive_numbers.append(num)
    elif num < 0:
        negative_numbers.append(num)

    user_input = input("Enter number or 'stop': ")

print("posetive numbers:", positive_numbers)
print("negative numbers:", negative_numbers)

File: Level 101/Homework/HW6.js
function double(number){
    return number * 2
}

function square(number){
    return number * number
}

function negative(number){
    return -number
}

function processNumber(number, operation) {
    return operation(number)
}

console.log(processNumber(5, double))
console.log(processNumber(5, square))
console.log(processNumber(5, negative))

